spec/spec_helper.rb

```diff
@@ -29,14 +29,15 @@
   minimum_coverage_by_file 26
 end
 
-# Ensure all /lib files are loaded
-# so they will be included in the test coverage report.
-Dir['./lib/**/*.rb'].sort.each { |file| require file }
-
 require 'faraday'
```

Comment:
Just as a note: this should not work correctly with global installed `faraday`. We should use `require_relative` here. But it's for a separate PR.

Reply:
I believe this shouldn't be an issue if you run `bundle exec rspec`, but yeah I agree this can be discussed separately 🙂